package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "node.js_course_practice",
+  "version": "1.0.0",
+  "description": "",
+  "main": "src/index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "nodemon index.js",
+    "start:dev": "nodemon --watch src"
+  },
+  "author": "",
+  "license": "ISC",
+  "devDependencies": {
+    "express": "^4.18.2",
```

Comment:
nodemon should be in the devDependencies section as it is using only for local development, others need to be as dependencies